packages/botonic-plugin-ner/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "@botonic/plugin-ner",
+  "version": "0.18.0",
+  "license": "MIT",
+  "main": "dist/index",
+  "scripts": {
+    "clean_install": "npm i && npm run-script build",
+    "cloc": "../../scripts/qa/cloc-package.sh .",
+    "build": "rm -rf dist && ../../node_modules/.bin/tsc",
+    "test": "jest",
+    "test_ci": "../../node_modules/.bin/jest --coverage --ci --reporters=default --reporters=jest-junit",
+    "prepare": "node ../../preinstall.js",
+    "lint": "npm run lint_core -- --fix",
+    "lint_ci": "npm run lint_core -- -c ../.eslintrc_slow.js",
+    "lint_core": "../../node_modules/.bin/eslint_d --cache --quiet 'src/**/*.ts*'"
+  },
+  "dependencies": {
+    "@babel/runtime": "^7.5.5",
+    "@botonic/core": "~0.18.0",
+    "@types/webgl2": "0.0.5",
+    "axios": "latest",
+    "node-fetch": "^2.6.1",
+    "ts-node": "^9.0.0",
```

Comment:
move @types/* to devdependencies as they are not necessary for runtime
remove typescript and ts-node because they're already in parent folder's package.json